app/scripts/components/granules/overview.js

```diff
@@ -89,6 +95,10 @@ class GranulesOverview extends React.Component {
     return applyWorkflowToGranule(granuleId, this.state.workflow);
   }
 
+  applyRecoveryWorkflow (granuleId) {
```

Comment:
How does `granuleId` get passed in here?

Reply:
I don't know concretely. It definitely has something to do with https://github.com/nasa/cumulus-dashboard/blob/master/app/scripts/components/table/list-view.js#L206 `BatchAsyncCommand`s... I'll do some more digging.

Reply:
Hmm okay. Yeah I didn't realize this was just part of how bulk actions work on the dashboard in general. I still don't see where the argument gets passed in, but it's not important to figure out right now

Reply:
It looks like the code automatically passes the ID associated with the row(s) you selected (I'm not sure how it determines the ID) to the action: https://github.com/nasa/cumulus-dashboard/blob/master/app/scripts/components/form/batch-async-command.js#L79

So yeah, I wouldn't looking into it more